lab/manifests/system/base/ironic/internal/base/kustomization.yaml

```diff
@@ -16,5 +18,15 @@ configMapGenerator:
   - name: ironic-inspector-conf
     files:
       - ironic-inspector.conf
+  - name: ironic-conf-overrides
+    literals:
+      - OS_CONDUCTOR__BOOTLOADER=HTTP_URL/uefi_esp.img
```

Comment:
I usually use underscores in fields expected to be rewritten by replacement transformer: underscored are prohibited in DNS domain names (except SRV records), so if some replacement didn't fail then everything will fail earlier (in some cases like Ingress even on `kubectl apply`)